home/models.py

```diff
@@ -340,6 +341,29 @@ def whatsapp_template_name(self):
     def whatsapp_template_body(self):
         return self.whatsapp_body.raw_data[0]["value"]["message"]
 
+    def clean(self):
+        message_with_media_length = 1024
+        errors = []
+        for message in self.whatsapp_body:
+            if (
+                (
+                    "image" in message.value
+                    and "document" in message.value
+                    and "media" in message.value
```

Comment:
Will all these fields be in `value`, even if there's only a `document`, will the `image` and `media` fields also be present?

Reply:
When the import PR is merged in, I would like to take this out, at the moment the import doesnt create objects under those 3 keywords, it only creates for `"image"` but if `"image"` is present in the block and the other two arent, the clean fails. This is just a work-around for the tests and using the import. But pages created by hand, and pages created with the updated import will have all three objects in the block